### DISCUSSION TITLE
Section 3: Unable to locate the .pandoc/ directory so this what i did

### DISCUSSION THREAD

**Discussion by kwamentw** [Cyfrin/security-and-auditing-full-course-s23#15] [Ideas]
i was unable to find the .pandoc/ so i downloaded the file from Patricks repo to my downloads folder
then ran the main command that creates the pdf this way.

```bash
pandoc report.md -o report.pdf --from markdown --template=~/Downloads/eisvogel --listings
```

works for me i dont know if it would for you!

Note: You need to have the Logo.pdf file in your audit-data directory to have a successful execution

**Comment by cromewar** [comment]
Nice @kwamentw 
You are nailing it, amazing!

**Reply by kwamentw** [reply]
Glad I could help you out!

**Comment by BowTiedBrothers** [comment]
The only thing I had to change was the directory slightly to: "--template=../../../Downloads/eisvogel --listings" and it worked for me.

Thank you!

**Reply by kwamentw** [reply]
I'm glad it did do well to Upvote so more people can find this

**Comment by rube-de** [comment]
I also ran into this. If you want to use just the template name, here is the correct way:

1. Get the user data directory for pandoc by running `pandoc --version` (for me on mac this was ~/.local/share/pandoc)
2. If the dir exists good, else create the pandoc and templates folders.
3. Put the eisvogel.latex in templates folder

(you should have now the file at ~/.local/share/pandoc/templates/eisvogel.latex )

**Reply by DuncanDuMond** [reply]
I was able to find the folder, placed the eisvogel latex file in that folder, and get this prompt:
```
Could not find data file templates/eisvogel.latex
```

This is where I placed the file:
<img width="358" alt="Screenshot 2024-01-01 at 8 48 49 PM" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/71061346/88ea8f53-cba5-44b6-a2db-bfbdc149dd02">

Please advise

**Reply by rube-de** [reply]
It has to go into pandoc/template.
If you don't have the template folder you need to create it.
(I updated make comment above for clarity)

**Reply by DuncanDuMond** [reply]
So I created the folder and still get the same error prompt. 

I even tried this prompt
```
pandoc report.md -o report.pdf --from markdown --template=~/.local/share/pandoc/template/eisvogel --listings
```
but still get
```
Could not find data file templates/eisvogel.latex
```

I must have input the wrong prompt even though I tried several derivates of 
```
pandoc report-example.md -o report.pdf --from markdown --template=eisvogel --listings
```

**Reply by rube-de** [reply]
Missed a S on templates, it's pandoc/templates/

**Reply by DuncanDuMond** [reply]
FINALLY!!!! I got it to work! Phew! Thanks a bunch!

**Reply by kwamentw** [reply]
Finally! 🔥

**Reply by AnouarBF** [reply]
It showed me another error! 
```
Error producing PDF.
! Missing number, treated as zero.
<to be read again> 
                   \protect 
l.335 \vspace\*
```
here is the command:
```
pandoc report.md -o report.pdf --from markdown --template=~/.pandoc/templates/eisvogel.latex
```
clearly the command is working now, but where this came from ?

**Reply by kwamentw** [reply]
Try going through your final file or if possible start the whole process again by using a new template, add your findings and regenerate the PDF

**Reply by AnouarBF** [reply]
do you know any other tool that could work rather than pandoc ?? It seems not working for me

**Reply by kwamentw** [reply]
Naah bro, I'd advise you don't give up cos we will encounter same obstacles on our auditing journey. @DuncanDuMond didn't give up & it worked

**Reply by AnouarBF** [reply]
No of course I'm not giving up! I'm just seeking for an alternative solution till the pandoc works, I don't want to be blocked on this step.

**Reply by GalBarZakay** [reply]
Hi, running into the same issue... Any suggestions? :)

**Reply by kwamentw** [reply]
https://github.com/Cyfrin/security-and-auditing-full-course-s23/discussions/15#discussion-5941217

Have you followed the steps here?

**Reply by kwamentw** [reply]
@GalBarZakay try repeating the whole process all over again and let's see pay attention to the little details tho

**Reply by Parth-Sharma-tonchi** [reply]
Do you find any solution to this problem? I am getting the same error: $ pandoc report.md -o report.pdf --from markdown --template=~/.pandoc/templates/eisvogel.latex
Error producing PDF.
! Missing number, treated as zero.
<to be read again> 
                   \protect 
l.335 \vspace\*

**Reply by kwamentw** [reply]
I recommend trying the whole process all over again maybe you left something out, you didn't notice

**Reply by garyghayrat** [reply]
What worked for me was making sure the `pandoc` metadata relevant part of the code is copied over as it's formatted in [`report-example.md`](https://github.com/Cyfrin/audit-report-templating/blob/aef38d7cd43b0334eb93817f01be787ed998195b/report-example.md) and then **Save without formatting** either by opening the Command Palette using `Shift+Command+P` and searching for `Save without formatting` or by using the keyboard shortcut `Command+K` then pressing `S`.

**Comment by kwamentw** [comment]
Okok you really have to get going
